functions/sum-of-inputs.py

```diff
@@ -0,0 +1,18 @@
+
+def sum_of_inputs(a,b):           # To calculate the sume of given value.
+		c=a+b
+		return c
+
+def find_type_arg(input1,input2): # To find the type of arguments given.
+	x,y=type(input1),type(input2)
```

Comment:
you should not compare two values, instead you should check both arguments or integer or string. Suppose if i pass both argument other than int or string then it should  return -1.